client/my-sites/theme/main.jsx

```diff
@@ -885,20 +885,31 @@ class ThemeSheet extends Component {
 		}
 
 		if ( hasWpOrgThemeUpsellBanner || hasThemeUpsellBannerAtomic ) {
+			const thisPageUrl = `/theme/${ themeId }/${ siteSlug }`;
 			pageUpsellBanner = (
 				<UpsellNudge
 					plan={ PLAN_BUSINESS }
 					className="theme__page-upsell-banner"
-					title={ translate( 'Access this theme for FREE with a Business plan!' ) }
+					title={
+						siteId
+							? translate( 'Upgrade your plan to access third-party themes' )
+							: translate( 'Access third-party themes with a Business plan' )
+					}
 					description={ preventWidows(
 						translate(
-							'Instantly unlock thousands of different themes and install your own when you upgrade.'
+							'Instantly unlock thousands of different themes and install your own when you upgrade to the Business plan.'
 						)
 					) }
 					forceHref
 					feature={ FEATURE_UPLOAD_THEMES }
 					forceDisplay
-					href={ ! siteId ? '/plans' : null }
+					href={
+						siteId
+							? `/checkout/${ siteSlug }/business?redirect_to=${ encodeURIComponent(
+									thisPageUrl
+							  ) }`
+							: localizeUrl( 'https://wordpress.com/pricing' )
+					}
```

Comment:
Should this link to `/start/business` instead, similarly to the CTA displayed when there are no results in the theme showcase?

https://github.com/Automattic/wp-calypso/blob/7c125bd5721a55574a0bc3768fb7451b796225f7/client/components/themes-list/index.jsx#L255

Reply:
That's a better idea yes 👍 

I think ideally when logged out then you'd go to /start/business if you sign-up and possibly to choose a site and checkout if you log-in. That's a different kettle of fish I think though, so lets go with this.